Session/Notifications/AppNotifications.swift

```diff
@@ -384,9 +384,12 @@ public class NotificationPresenter: NSObject, NotificationsProtocol {
             closedGroupName: nil,       // Not supported
             openGroupName: nil          // Not supported
         )
+        
+        let fallbackSound: Preferences.Sound = db[.defaultNotificationSound]
+            .defaulting(to: Preferences.Sound.defaultNotificationSound)
 
         DispatchQueue.main.async {
-            let sound = self.requestSound(thread: thread)
+            let sound = self.requestSound(thread: thread, fallbackSound: fallbackSound)
```

Comment:
Unrelated changes?

Reply:
Can't compile without this change after merging dev branch.

Reply:
Yep just noticed this - sorry about that